webapp/components/Indicators.jsx

```diff
@@ -17,10 +17,7 @@ class Indicators extends Component {
   render() {
     return (
       <div>
-        <TransitionGroup
-          transitionName="toast"
-          transitionEnter={false}
-          transitionLeaveTimeout={500}>
+        <TransitionGroup enter={false}>
```

Comment:
i think this is not behaving the same as before, but i gave up after 10 minutes. it at least works mostly ok